.github/workflows/main/installDelight.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+import shutil
+
+if sys.version_info[0] < 3 :
+	from urllib import urlretrieve
+else :
+	from urllib.request import urlretrieve
+
+delightVersion="1.1.12"
+delightDirectory="free/beta/2018-11-01-oIDoJTpO"
+
+baseUrl = "https://3delight-downloads.s3-us-east-2.amazonaws.com"
+
+if sys.platform == "linux" :
```

Comment:
I think this may be the reason why 3delight isn't getting installed. In python 2, `sys.platform == "linux2"`.

Reply:
Thanks! Addressed in c6527f89dd90108e87df35d87ce386298c214d01